DemoqaProject/pageObjects/Forms/PracticeForm.cs

```diff
@@ -0,0 +1,145 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.PageObjects;
+
+namespace DemoqaProject.PageObjects
+{
+    public class PracticeForm : Forms
+    {
+        private string YEAR = "1994";
+        private string MONTH = "July";
+        private static string? RunningPath = Directory.GetParent(Directory.GetParent(Directory.GetParent(Environment.CurrentDirectory).ToString()).ToString())?.ToString();
+        public string uploadURL = Path.GetFullPath(Path.Combine(RunningPath ?? string.Empty, $"files{Path.DirectorySeparatorChar}sampleFile.jpeg"));
+
+        [FindsBy(How = How.Id, Using = "firstName")]
+        public IWebElement firstName;
+        
+        [FindsBy(How = How.Id, Using = "lastName")]
+        public IWebElement lastName;
+        
+        [FindsBy(How = How.Id, Using = "userEmail")]
+        public IWebElement userEmail;
+        
+        [FindsBy(How = How.XPath, Using = "//label[@for='gender-radio-1']")]
+        public IWebElement male;
+        
+        [FindsBy(How = How.Id, Using = "userNumber")]
+        public IWebElement userNumber;
+        
+        [FindsBy(How = How.Id, Using = "dateOfBirthInput")]
+        public IWebElement dateOfBirth;
+
+        [FindsBy(How = How.ClassName, Using = "react-datepicker__month-select")]
+        public IWebElement pickMonth;
+        
+        [FindsBy(How = How.ClassName, Using = "react-datepicker__year-select")]
+        public IWebElement pickYear;
+        
+        [FindsBy(How = How.XPath, Using = "//div[@class='react-datepicker__day react-datepicker__day--024 react-datepicker__day--weekend']")]
+        public IWebElement pickDate;
+
+        [FindsBy(How = How.Id, Using = "subjectsInput")]
+        public IWebElement subject;
+
+        [FindsBy(How = How.XPath, Using = "//div[@class='custom-control custom-checkbox custom-control-inline']")]
+        public IList<IWebElement> hobbies;
+
+        [FindsBy(How = How.Id, Using = "uploadPicture")]
+        public IWebElement uploadPhoto;
+
+        [FindsBy(How = How.Id, Using = "currentAddress")]
+        public IWebElement currentAddress;
+        
+        [FindsBy(How = How.Id, Using = "react-select-3-input")]
+        public IWebElement state;
+        
+        [FindsBy(How = How.Id, Using = "react-select-4-input")]
+        public IWebElement city;
+        
+        [FindsBy(How = How.Id, Using = "submit")]
+        public IWebElement submit;
+
+        [FindsBy(How = How.ClassName, Using = "modal-content")]
+        public IList<IWebElement> tableName;
+
+
+        public void FillFormWithValidCredentials(string firstname, string lastname, string useremail, string usernumber, string subj, string address, string state1, string city1)
```

Comment:
1) You have two identical methods, do not repeat yourself.
2) Method is way too big.
3) Decompose this method into smaller actions, like typing picking date, selecting hobbies, uploading photo and etc. Create a method that would combine these small methods.

Additionally, your method has too many parameters to take. There is an option to create a model for such object and do a parametrized test providing this object as an argument for this method in test. But for now it's ok to do it that way.